howto/regex.po

```diff
@@ -32,160 +32,250 @@ msgstr ""
 #: ../Doc/howto/regex.rst:18
 msgid ""
 "This document is an introductory tutorial to using regular expressions in "
-"Python with the :mod:`re` module.  It provides a gentler introduction than "
-"the corresponding section in the Library Reference."
+"Python with the :mod:`re` module.  It provides a gentler introduction than the "
+"corresponding section in the Library Reference."
 msgstr ""
+"Ce document est un tutoriel d'introduction à l'utilisation des expressions "
```

Comment:
Maybe use "guide" instead of "tutoriel" if we choose "guide" in place of "howto" for consistency.

Reply:
Done